libvast/src/system/exporter.cpp

```diff
@@ -154,11 +162,15 @@ behavior exporter(stateful_actor<exporter_state>* self, expression expr,
           self->state.results.push_back(std::move(candidate));
         else
           VAST_DEBUG(self, "ignores false positive:", candidate);
-        mask.append_bits(false, candidate.id() - mask.size());
-        mask.append_bit(true);
+        if (sender == self->state.index || sender == self->state.archive) {
+          mask.append_bits(false, candidate.id() - mask.size());
+          mask.append_bit(true);
+        }
+      }
+      if (sender == self->state.index || sender == self->state.archive) {
+        self->state.stats.processed += candidates.size();
+        self->state.unprocessed -= mask;
```

Comment:
Why should we ignore the bookkeeping of processed hits for continuous queries? Performance?

I could imagine that we want to set `unprocessed` to an all-ones bitmap. Then we wouldn't need to touch this logic at all.